### PR TITLE
fix instructions for running install.sh

### LINKED ISSUE #11: install.sh script doesn't work on Ubuntu
### Current Behavior

I followed the instruction in README.md but I encountered the following error. The script expects bash but the script uses another shell. I thought that the correct command is `./install.sh` instead of `sh ./install.sh`.

```
$ sh ./install.sh
**************************************************************
 FlowForge Installer

 Warning:
 If this script needs to install NodeJS it may ask for your
 password in able to run some commands as root.

**************************************************************
NodeJS found
./install.sh: 55: [[: not found
**************************************************************
 You need NodeJS 16 or newer, please upgrade
**************************************************************
```

### Expected Behavior

`./install.sh` works correctly.
```
$ ./install.sh
**************************************************************
 FlowForge Installer

 Warning:
 If this script needs to install NodeJS it may ask for your
 password in able to run some commands as root.

**************************************************************
NodeJS found
**************************************************************
 NodeJS Version 16 or newer found
**************************************************************
**************************************************************
 npm found
**************************************************************
8.1.2
**************************************************************
 Installing FlowForge
**************************************************************
⸨⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⸩ ⠦ idealTree:app: sill idealTree buildDeps
```

### Steps To Reproduce

_No response_

### Environment

- FlowForge version: v0.1.0
- Node.js version: v16.13.1
- npm version: v8.1.2
- Platform/OS: Ubuntu 20.04.2 LTS on WSL2 in Windows 10
- Browser: Google Chrome 97.0.4692.99 (Official Build)(64 bit)

### LINKED ISSUE #12: Incorrect directory name inside the installer zip file
### Current Behavior

When I tried the instructions in README.md, I found that `cp -R flowforge-installer-0.1.0/* /opt/flowforge` command doesn't work.

```
$ cp -R flowforge-installer-0.1.0/* /opt/flowforge
cp: cannot stat 'flowforge-installer-0.1.0/*': No such file or directory
```

The problem occurs from wrong directory name inside the installer zip file. It has `flowforge-install-0.1.0` directory.

image

### Expected Behavior

The command executes without error.

### Steps To Reproduce

_No response_

### Environment

- FlowForge version: v0.1.0
- Node.js version: v16.13.1
- npm version: v8.1.2
- Platform/OS: Ubuntu 20.04.2 LTS on WSL2 in Windows 10
- Browser: Google Chrome 97.0.4692.99 (Official Build)(64 bit)

### PR DESCRIPTION
**Pull request by hardillb**
fixes #11  and fixes #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,14 +73,14 @@ xcode-select --install
    ```
     cd /tmp/
     unzip flowforge-installer-x.y.z.zip
-    cp -R flowforge-installer-x.y.z/* /opt/flowforge
+    cp -R flowforge-instal-x.y.z/* /opt/flowforge
```
Comment (kazuhitoyokoi) [inline README.md:76]: `instal` -> `install` if the document will be changed.

**Comment by knolleary** [comment]
I think we want it to be called '-installer' but okay to fixup the docs to match what we have, and change back for 0.2

**Comment by hardillb** [comment]
yep, went for the easiest fix for now.

**Comment by kazuhitoyokoi** [comment]
Thank you for fixing it. I will try to use FlowForge more. :)

**Pull request merged**